### PR TITLE
add support for tengine uint8

### PR DESCRIPTION
**Pull request by un-knight**
Add support for tengine uint8 backend, mainly refers to:
- https://github.com/OAID/Tengine/blob/45863aa6d928f5967bf05811cada6a166a20ca91/tools/convert_tool/utils/graph_optimizer/graph_opt.cpp#L931
- https://github.com/OAID/Tengine/blob/45863aa6d928f5967bf05811cada6a166a20ca91/tools/quantize/quant_save_graph.cpp#L38
- https://github.com/OAID/Tengine/blob/45863aa6d928f5967bf05811cada6a166a20ca91/tools/quantize/quant_tool_uint8.cpp#L89

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
-torch==1.8.1
-torchvision==0.9.1
+# torch==1.8.1
```
Comment (Tracin) [inline requirements.txt]: Not compatible with torch v1.81?

**Comment by Tracin** [comment]
We hightly recommend to add few test cases for this backend !

File: requirements.txt (hunk 1 above)
Comment (un-knight) [reply requirements.txt]: nope, since my developed environment is torch 1.9.0, I remove the requirement of PyTorch 1.8.1 temporarily. But this tengine expansion can be totally compatible with torch v1.8.1, and I will restore the requirements.

**Comment by un-knight** [comment]
> We hightly recommend to add few test cases for this backend !

sure, it's necessary to add test cases, but what kind of test cases do we need? Will a table of metrics about imagenet's accuracy with a model before and after quantization be a helper?

**Comment by Tracin** [comment]
> > We hightly recommend to add few test cases for this backend !
> 
> sure, it's necessary to add test cases, but what kind of test cases do we need? Will a table of metrics about imagenet's accuracy with a model before and after quantization be a helper?

Something like https://github.com/ModelTC/MQBench/blob/main/test/backend/test_backend.py will be enough for now.
Any table of metrics on datasets can be very convincing if that is possible !

**Comment by un-knight** [comment]
There is MobileNetV2's accuracy on the ImageNet-1k classification task. Since full training of ImageNet is time-consuming, I just finetune the model for several steps, and this is why the top1 accuracy is lower than expected.

| model | top1@QAT_tengine_u8 | top1@tengine_u8_x86 | 
| ---- | --------| --------|
| mobilenet_v2 | 55.06 | 54.862 |

**Comment by XHPlus** [comment]
Thanks for the awesome PR and looking forward to your continuous contribution! @un-knight
